ui.content/src/main/content/jcr_root/content/_cq_graphql/global/endpoint/.content.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:primaryType="nt:unstructured"
+    sling:resourceType="graphql/sites/components/endpoint"
+    sling:redirect="{Boolean}false"
+    sling:vanityOrder="{Long}301"
```

Comment:
Is this supposed to be `sling:redirectStatus` or just a coincidence the vanity order is the same as a perm redirect?